### PR TITLE
doc: update news

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/news.rst [hunk 1]
```diff
@@ -7,6 +7,79 @@
...
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/executables/grndb`][:ref:`grndb-force-truncate`] Improved
+  ``grndb recover --force-truncate`` option that it can be truncated even if
+  locks are left on the table.
+
+* [Windows] Use ``CONDITION_VARIABLE`` instead of Groonga's original implementation.
+
```
Comment (kenhys) [inline doc/source/news.rst]: It's too close to internal implementation, what is the reason for implementation replacement and what's the benefit from? It seems that "why replaced" and "it solves ..." is important.

File: doc/source/news.rst [hunk 2]
```diff
@@ -7,6 +7,79 @@
...
+* [:doc:`/reference/commands/logical_range_filter`] Added ``sort_keys`` option.
+
+* Added a new function ``time_format()``.
+  You can specify time format against a column of Time type.
+  You can specify with use format of ``strftime`` .
+
```
Comment (kenhys) [inline doc/source/news.rst]: Add a link to `time_format()` documentation is good if you afford to do.
`Time` is better to markup as <pre>\`Time\`</pre>

File: doc/source/news.rst [hunk 3]
```diff
@@ -7,6 +7,79 @@
...
+  You can dynamically change tokenizer's behavior.
+  Specific options is as below.
+  * ``n`` : You can specify "N" of Ngram. For example, Ngram is trigram
+    when you can set "3" to "N".
+  * ``loose_symbol`` : The keywords including symbols are hit, even the
+    keywords not including symbols are hit also.
+  * ``remove_blank`` : Ignore blank.
+  * ``loose_blank`` : The keywords including blank are hit, even the
+    keywords not including blank are hit also.
+
```
Comment (kenhys) [inline doc/source/news.rst]: Could you add some example case for above options? For example, phone number or something else.

File: doc/source/news.rst [hunk 4]
```diff
@@ -7,6 +7,79 @@
...
+* [:doc:`/reference/normalizers`] Support Unicode 10.0
+
```
Comment (kenhys) [inline doc/source/news.rst]: Could you refer to `NormalizerNFKC100`?

File: doc/source/news.rst [hunk 5]
```diff
@@ -7,6 +7,79 @@
...
+  normalizer's options.
+
+* [httpd] Updated bundled nginx to 1.13.12.
+
+Fixes
+^^^^^
+
+* Fixed a bug that unexpected record is matched with space only query.
+  [groonga-dev,04609][Reported by satouyuzh]
+
```
Comment (kenhys) [inline doc/source/news.rst]: It seems that https://github.com/groonga/groonga/commit/e8203c5fe805fcb4d2aebab3a6fdc5b628d27178 entry is missing from news.

Comment (kenhys) [reply doc/source/news.rst]: Ubuntu 18.04 support is also missing?

Comment (kenhys) [reply doc/source/news.rst]: It seems that msgpack-c < 1 was dropped. am i wrong?

File: doc/source/news.rst (hunk 1 above)
Comment (komainu8) [reply doc/source/news.rst]: I changed to a description as below.
```
Fixed a bug that it may also take so much time to change "thread_limit".
```

**Comment by komainu8** [comment]
@kenhys Thank you for your comments.
I've fixed them.

File: doc/source/news.rst (hunk 5 above)
Comment (kou) [reply doc/source/news.rst]: It's not dropped. If Groonga isn't buildable with msgpack-c < 1, it's a bug.

Comment (kenhys) [reply doc/source/news.rst]: Thanks, I got it.

**Comment by kenhys** [comment]
@komainu8  Merged.

**Pull request merged**